src/layouts/components/Header.tsx

```diff
@@ -42,6 +42,7 @@ export function Header(): JSX.Element {
 
     router.events.on("routeChangeStart", routeChangeStart);
     return () => router.events.off("routeChangeStart", routeChangeStart);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
is this required? I think eslint is only throwing warning for any violation of this error

Reply:
To remove that warning only I have added this